### DISCUSSION TITLE
reComputer for Jetson Initiation | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#138] [Q&A] [answered]
# reComputer for Jetson Initiation | Seeed Studio Wiki

reComputer for Jetson Initiation

https://wiki.seeedstudio.com/reComputer_Jetson_Series_Initiation/

**Comment by l43lu** [comment]
Couple of interesting links are just dead:
https://wiki.seeedstudio.com/Tutorial-of-A20X-Carrier-Boards/#flashing-to-emmc-with-sdk-manager
https://wiki.seeedstudio.com/Tutorial-of-A20X-Carrier-Boards/#flashing-to-emmc-with-command-line

etc..

**Reply by MatthewJeffson** [reply]
Hi, thank you for pointing it! I will confirm it and then change them :D

**Comment by fraborg99** [comment]
Hi, the links to install the NVIDIA system are not working.

**Reply by MatthewJeffson** [reply, marked as answer]
Hi, we have divided the previous wiki into several parts, thus the links are broken. The steps to install the system can be found in the left column, right below this article "recomputer > Jetpack OS Installation" and "reComputer > Carrier Board for Jetson Module" in the left column. <br/> I will update this wiki later. Thank you for your interests!

**Comment by harshashejwal** [comment]
Hello,

Please help me my recomputer J2021 ki not working, when I power on it only show the Nvidia logo ,not showing OS windods ,only showing some type of error these some instruction as like 6.3529021 CPU4: shutdown, 7.236921 Bridge firewalling registered as like many error are showing ,Please help me out form this problem , how to solve it.

**Reply by MatthewJeffson** [reply]
Hello! Have you tried to [flash back the JetPack OS on reComputer J2021](https://wiki.seeedstudio.com/reComputer_J2021_J202_Flash_Jetpack/)?
